go.mod

```diff
@@ -4,19 +4,28 @@ go 1.14
 
 require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
-	github.com/ably-forks/boomer v0.0.0-20200717132144-47e73943070d
-	github.com/ably/ably-go v1.1.6-0.20200905151113-60928e378b37
+	github.com/ably-forks/boomer v0.0.0-20210109192149-bcdec6481b67
```

Comment:
Looks like this points to a branch - perhaps we should consider merging into master?

https://github.com/myzhan/boomer/compare/master...ably-forks:local-spawn-event

Reply:
Yeh I opened an upstream PR https://github.com/myzhan/boomer/pull/113, hoping that will get merged and we can drop the fork.

Reply:
Ah nice, I didn't see that. Dropping the fork will be good 👍